utils.py

```diff
@@ -496,7 +427,7 @@ def cli_download(uri, destination):
     # Download from uri in thread while showing progress bar.
     cli_queue = queue.Queue()
     args = [uri, target]
-    kwargs = kwargs={'q': cli_queue}
+    kwargs = kwargs = {'q': cli_queue}
```

Comment:
Not sure how we got `kwargs` defined twice, but this just needs to be ` kwargs = {'q': cli_queue}`